### PR TITLE
Evan/test no record serialisation

### PR DESCRIPTION
**Pull request by evantahler**
Similar to https://github.com/airbytehq/airbyte/pull/16995, this PR may make Airbyte more faster maybe.  This attempt stores a copy of the original, still-serialized message on all AirbyteMessages so we don't need to spend CPU time re-serializing them.

**[Learn more in the doc](https://docs.google.com/document/d/1z7ngxncrX44inDnGnBQJQvkCKew_4MQb2Fr_slxzGoI/edit#)!**

Some background on this:
* https://github.com/airbytehq/airbyte/issues/12532#issuecomment-1247412394
* https://airbytehq.slack.com/archives/C019WJFR1JT/p1622103941088300

### CODE REVIEW

**Comment by evantahler** [comment]
Should we choose to move forward with this approach to trade memory usage to skip a serialization... what's the best way to test it before rolling it out?

**Comment by davinchia** [comment]
@evantahler I thought I commented here.. I was wondering, what's the result of this approach? Have we confirmed this has awesome CPU effects and similar memory effects?

File: airbyte-workers/src/main/java/io/airbyte/workers/internal/DefaultAirbyteStreamFactory.java [hunk 1]
```diff
@@ -61,6 +63,8 @@ public Stream<AirbyteMessage> create(final BufferedReader bufferedReader) {
             try (final var mdcScope = containerLogMdcBuilder.build()) {
               logger.info(line);
             }
+          } else if (!jsonLine.get().isEmpty()) {
+            ((ObjectNode) jsonLine.get()).put("serialized_message", line);
```
Comment (evantahler) [inline airbyte-workers/src/main/java/io/airbyte/workers/internal/DefaultAirbyteStreamFactory.java:67]: This feels like a type hack and I have no idea what I'm doing.

**Comment by evantahler** [comment]
@davinchia using the same test suite that confirmed https://github.com/airbytehq/airbyte/pull/16995 was _clearly_ better, this PR is _slightly_ better, and at best the same.  I do not think we should peruse it at this moment.

**Comment by davinchia** [comment]
> at best the same
 
make sure I understand - the approach in this PR is slight better -> equal to master?

strange. i would imagine holding onto the message in memory is fine

**Comment by evantahler** [comment]
I think what I learned from this PR is that JSON parsing is slow/expensive by serialization on the way out is rather fast/easy - this PR only replaced the serialization part of the worker.

**Pull request closed without merging**